python/nano/src/bigdl/nano/tf/keras/inference/_worker.py

```diff
@@ -45,5 +47,10 @@ def throughput_calculate_helper(iterrun, func, model, input_sample):
         tf.config.threading.set_inter_op_parallelism_threads(0)
         tf.config.threading.set_intra_op_parallelism_threads(thread_num)
     params = pickle.load(open(param_file, "rb"))
```

Comment:
may use `cloudpickle` to load because you saved it by `cloudpickle`

Reply:
Done